### ISSUE TITLE
Consider interpolating `${null}` as empty string

### ISSUE THREAD

**Issue by jonasfj** [taskcluster/json-e#154]
Currently we don't allow interpolation of objects and arrays...

these seems to work:
* `${"string"}` => `"string"`
* `${55.5}` => `"55.5"`
* `${null}` => `"null"`

I propose that we change it so that: `${null}` => `""`

It's weird that `null` becomes `"null"` when objects and arrays aren't allowed.
It would seems useful to allow `null` to be used as empty string, since that just means it disappears.

Example: `"${firstName} ${middleName} ${lastName}"` if `middleName` is `null` it'll still work... as is it just becomes the `"James null Bond"`, instead of `"James Bond"`

**Comment by djmitche** [comment]
This makes sense to me.  We should add a bit to the README too.

**Comment by alisha17** [comment]
Can I take up this issue to work upon?

**Comment by jonasfj** [comment]
@alisha17, absolutely...

Btw, we hang out on IRC in #taskcluster and #tc-contributors on irc.mozilla.org if you have any questions..

**Comment by alisha17** [comment]
@jonasfj  Could you guide me a bit about which JS file does this code relate to? Maybe interpreter.js?

**Comment by djmitche** [comment]
You probably want to start [here](https://github.com/taskcluster/json-e/blob/f63ffdcb269be0fbc2101ecde6facc011ae54c8b/src/index.js#L16-L45).

**Comment by alisha17** [comment]
Done! Regarding the test : [link](https://github.com/taskcluster/json-e/blob/master/test/specification_test.js#L16), where can I find this SPEC_FILE?
@djmitche

**Comment by djmitche** [comment]
In Node, `__dirname__` is the name of the directory containing the current source file (sort of like `__file__` in Python).  So

```
const SPEC_FILE = path.join(__dirname, '../specification.yml');
```

means that it's the file `specification.yml` in the parent directory to that containing `specification_test.js`.

**Comment by djmitche** [comment]
Fixed in #178.  @alisha17 by the way if you write something like "Fixes #154" in your pull request description, then Github will automatically close that issue when the PR is merged.

**Comment by alisha17** [comment]
@djmitche  Ah! Wasn't aware about it. Will take care now.
